e2e-tests/default-cr/run

```diff
@@ -48,26 +48,46 @@ function main() {
 
 	desc 'create secrets and start client'
 	kubectl_bin apply -f $deploy_dir/secrets.yaml
-	kubectl_bin apply -f $conf_dir/client.yml
+	apply_client $conf_dir/client.yml
 
 	desc "create first PSMDB cluster $cluster"
 	kubectl_bin apply ${OPERATOR_NS:+-n $OPERATOR_NS} --server-side --force-conflicts -f $deploy_dir/crd.yaml
+
+
+ 	local temp_operator_yaml="$(mktemp)"
+
 	if [ -n "$OPERATOR_NS" ]; then
 		apply_rbac cw-rbac
 		kubectl_bin apply -n ${OPERATOR_NS} -f $deploy_dir/cw-operator.yaml
 	else
 		apply_rbac rbac
-		yq eval '((.. | select(.[] == "DISABLE_TELEMETRY")) |= .value="true")' "$deploy_dir/operator.yaml" \
-			| kubectl_bin apply -f -
+		yq eval '((.. | select(.[] == "DISABLE_TELEMETRY")) |= .value="true")' "$deploy_dir/operator.yaml" > $temp_operator_yaml
+
+		if [[ "$ARCH" == "arm64" ]]; then
+			yq eval -i '(.spec.template.spec.tolerations += [{"key": "kubernetes.io/arch", "operator": "Equal", "value": "arm64", "effect": "NoSchedule"}])' $temp_operator_yaml
```

Comment:
Cant' we move this toleration value to some variable and use it for objects creation? If it changes at some moment we will need to update it in multiple places?
`[{"key": "kubernetes.io/arch", "operator": "Equal", "value": "arm64", "effect": "NoSchedule"}])'`

Reply:
Done, thanks!